tutorials/kitties-part-01/0-intro.mdx

```diff
@@ -0,0 +1,55 @@
+---
+tags:
+  - pallet
+  - poladot.js
+keywords: [pallet, polkadot-js]
+description:
+  From the substrate node template to a fully functional blockchain that handles kitties NFTs.
+level: intermediate
+duration: 2h
+---
+
+# How to build the Substrate Kitties Chain
+
+![kitties](./assets/kitties.png)
+
+Welcome to the Substrate Kitties tutorial part I. Designed to teach how to build a blockchain that handles 
+the creation and ownership management of NFT Kitties.
+
+It will cover various aspects of building the Kitties pallet, including creation, breeding, 
+transfer, price setting, and purchasing kitties. Additionally, it will provide guidance on developing the 
+front-end UI for interaction with the chain.
+
+This tutorial is an updated/adapted version of two previous tutorials, the [CryptoKitties on Substrate](https://www.shawntabrizi.com/substrate/cryptokitties-on-substrate/)
```

Comment:
Maybe add a line of code, why you updated it. is the others outdated? did you add features?